rosidl_runtime_c/include/rosidl_runtime_c/message_bounds_struct.h

```diff
@@ -28,21 +28,50 @@ typedef struct rosidl_message_bounds_t rosidl_message_bounds_t;
 typedef const rosidl_message_bounds_t * (* rosidl_message_bounds_handle_function)(
   const rosidl_message_bounds_t *, const char *);
 
+/// Contains rosidl message bounds data
 struct rosidl_message_bounds_t
 {
+  /// String identifier for the type_support.
   const char * typesupport_identifier;
+  /// pointer to type support handle function
```

Comment:
Capitalize any sentence at the start of a brief (`///`). Fix here and below.

Also, these both can't be pointers to the handle functions, describe what data is here